src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -84,6 +87,19 @@ public function getImpersonatePath(string $identifier): string
         return $this->impersonateUrlGenerator->generateImpersonationPath($identifier);
     }
 
+    public function userIsGranted(UserInterface $user, mixed $role, mixed $object = null, ?string $field = null): bool
```

Comment:
`role` is a bad name. Not all permission attributes are roles.

Reply:
and yes, I know that the argument name is bad for the `is_granted` Twig function. But this is not a reason to use a bad name for the new function (which does not have the same BC concern).

Reply:
This makes sense. Originally I was trying to match isGranted() as closely as possible, but it does make sense to use better naming.